components/ws-proxy/cmd/run.go

```diff
@@ -131,6 +131,25 @@ var runCmd = &cobra.Command{
 		}
 
 		// SSH Gateway
+
+		var caKey ssh.Signer
+		readCAKeyFile := func() {
+			caPrivateKeyB, err := os.ReadFile(cfg.Proxy.SSHGatewayCAKeyFile)
+			if err != nil {
+				log.WithError(err).Error("cannot read SSH Gateway CA key")
+				return
+			}
+			c, err := ssh.ParsePrivateKey(caPrivateKeyB)
+			if err != nil {
+				log.WithError(err).Error("cannot prase SSH Gateway CA key")
```

Comment:
@iQQBot should we return if we cannot parse the private key (similar to being unable to read the file)? This way, caKey is not set.

```suggestion
				log.WithError(err).Error("cannot parse SSH Gateway CA key")
				return
```